ui/DesignSystem/Component/Timeline.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+  import TimelineItem from "./Timeline/TimelineItem.svelte";
+  export let items = null;
+</script>
+
+<style>
+  ul {
+    margin-top: 24px;
+  }
+</style>
+
+<ul>
```

Comment:
I think this would make more sense with `timestamp`s that separate groups of `TimelineItem`s, e.g.

<img width="1176" alt="Screen Shot 2020-05-14 at 11 40 42 AM" src="https://user-images.githubusercontent.com/2712962/81973044-6947a900-95d8-11ea-9c19-48d10d4d829d.png">

Reply:
The `Input` actually should exist outside of `Timeline` entirely, and live underneath it in the `Issue` component.

Reply:
Yeah this was good pushback. I still think a general purpose `Item` made sense, but made it a bit smarter and more DRY. See updated version

Reply:
From what I understand, any timeline would always have a comment box, so wouldn't it make sense to have it part of the timeline component. I mean I now abstract it out into it's own component. Does this already resolve your concern or do you still see it living outside of timeline?